app/assets/javascripts/app/users/user.js

```diff
@@ -5,36 +5,40 @@
     .module('council.users')
     .controller('EditUserCtrl', EditUserController);
 
-  function EditUserController(User, userId, DS) {
+  function EditUserController($state, User, userId) {
     var ctrl = this;
 
     User.find(userId).then(resetUser);
 
-    ctrl.changePassword = changePassword;
+    ctrl.updateUser = updateUser;
     ctrl.message = '';
     cleanPasswordFields();
 
-    function changePassword() {
-      if (ctrl.password !== ctrl.passwordConfirmation) return;
+    function updateUser() {
+      if (ctrl.password !== ctrl.passwordConfirmation)
+        return;
 
       ctrl.user.update({
+        username: ctrl.username,
```

Comment:
This manual work you're doing here could be avoided if you nested all the form's data in a new object under `ctrl`, for example named *data*.
So it would end up being:
```
ctrl.user.update(ctrl.data)
  .then(userUpdated, userNotUpdated);
```